src/components/pages/HomePage/HowPulsarWorks/HowPulsarWorks.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import s from './HowPulsarWorks.module.css'
+import ScreenTitle from '../ui/ScreenTitle/ScreenTitle';
+
+// SVGO breaks the illustration. To fix it, we import it as is.
+import illustrationDesktop from '!!raw-loader!./img/illustration-desktop.svg';
+import illustrationMobile from '!!raw-loader!./img/illustration-mobile.svg';
+
+import BookkeeperIcon from './img/bookkeeper.svg';
+import BrokersIcon from './img/brokers.svg';
+import ZookeeperIcon from './img/zookeeper.svg';
+import ProducerAndConsumerIcon from './img/producer-and-consumer.svg';
+import Slider from '@site/src/components/ui/Slider/Slider';
+
+const cards: CardProps[] = [
+  {
+    title: 'Producer & Consumer',
+    image: <ProducerAndConsumerIcon />,
+    children: (
+      <p>
+        A Pulsar client contains a consumer and a producer.
+        A Producer writes messages on a topic.
+        A consumer reads messages from a topic and acknowledges specific messages or all up to a specific message.
+      </p>
+    )
+  },
+  {
+    title: 'Apache Zookeeper',
+    image: <ZookeeperIcon />,
+    children: (
+      <p>
+        Pulsar and Bookkeeper use Apache Zookeeper to save metadata coordinated between nodes,
+        such as a list of ledgers per topic, segments per ledger, and mapping of topic bundles to a Broker.
+        It’s a cluster of highly available and replicated servers (usually 3).
+      </p>
+    )
+  },
+  {
+    title: 'Pulsar Brokers',
+    image: <BrokersIcon />,
+    children: (
+      <p>
+        Topics (i.e., partitions) are divided among Pulsar Brokers.
+        A broker receives messages for a topic and appends them to the topic’s active virtual file (a.k.a Ledger),
+        hosted on the Bookkeeper cluster. Brokers read messages from the cache (mostly) or Bookkeeper and dispatch them to the Consumers.
```

Comment:
```suggestion
        hosted on the Bookkeeper cluster. Brokers read messages from the cache (mostly) or BookKeeper and dispatch them to the consumers.
```
why capitalize these words? (broker, producer, consumer, ledger,...)

Reply:
I thought they were special entities; hence, they need capitalization, but I was wrong.